packages/block-editor/src/components/use-block-drop-zone/index.js

```diff
@@ -20,6 +20,9 @@ import {
 } from '../../utils/math';
 import { store as blockEditorStore } from '../../store';
 
+const THRESHOLD_DISTANCE = 30;
+const MINIMUM_HEIGHT_FOR_THRESHOLD = 120;
```

Comment:
> I guess it makes more sense to want to drop things inside an empty group block rather than around it 😄

That was my thinking. These are just arbitrary numbers for now, but in practice, I figured you probably wouldn't want the threshold for the top/bottom positions being active unless there was at least the same amount of room on the inside of the block to allow for "real" internal dragging. So `120` seemed like a reasonable default to go with for now as it's 4 * the threshold distance. Happy to change any of these values, of course!